DiceKeys/Views/Screens/AppMainView.swift

```diff
@@ -112,7 +113,7 @@ struct AppMainView: View {
                 }
             }.buttonStyle(PlainButtonStyle())
             Spacer()
-        }
+            }.frame(width: 480, height: 700)
```

Comment:
Adding the frame here makes sure macOS app always launch with same size

Reply:
Since this is shared code, shouldn't it be in conditional?  We don't want to force a size in iOS when there are such a large variety of screen sizes.

Reply:
Yeah makes sense